chatterbot/adapters/plugins/evaluate_mathematically.py

```diff
@@ -0,0 +1,183 @@
+from .plugin import PluginAdapter
+import re
+import os, json
+import decimal
+
+class EvaluateMathematically(PluginAdapter):
+
+    def should_answer(self, input_text):
+        """
+        Determines whether it is appropriate for this plugin
+        to respond to the user input.
+        """
+
+        response = self.process( input_text )
+
+        if response is False:
+            return False
+        else:
+            return True
+
+
+    def process(self, input_text):
+        """
+        Takes a statement string.
+        Returns the simplified statement string
+        with the mathematical terms "solved".
+        """
+
+        # Getting the mathematical terms within the input statement
+        expression = self.simplify_chunks( self.normalize( input_text ) )
+
+        # Returning important information
+        try:
+            expression += '= ' + str( eval( expression ) )
+
+            return expression
+        except:
+            return False
+
+
+    def simplify_chunks(self, input_text):
+        """
+        Separates the incoming text.
+        """
+
+        string = ''
+
+        for chunk in input_text.split( ' ' ):
```

Comment:
Should I use ```input_text.split()``` instead of ```input_text.split( ' ' )```? That way I would be separating the words based on all whitespace characters not just spaces.